libs/src/services/solanaWeb3Manager/rewards.js

```diff
@@ -527,6 +628,158 @@ const generateSecpInstruction = ({
   })
 }
 
+const generateCreateSenderSecpInstruction = ({
+  ethAddress,
+  attestationMeta,
+  rewardManagerAccount,
+  instructionIndex
+}) => {
+  // Perform signature manipulations:
```

Comment:
Looks like this is the same code as the other SECP instruction - could we pull this out into a helper? Something like `encodeSignature`?